src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php

```diff
@@ -37,7 +37,11 @@ public function process(ContainerBuilder $container)
             if (!is_subclass_of($class, 'Symfony\\Component\\Console\\Command\\Command')) {
                 throw new \InvalidArgumentException(sprintf('The service "%s" tagged "console.command" must be a subclass of "Symfony\\Component\\Console\\Command\\Command".', $id));
             }
-            $container->setAlias($serviceId = 'console.command.'.strtolower(str_replace('\\', '_', $class)), $id);
+            $serviceId = 'console.command.'.strtolower(str_replace('\\', '_', $class));
+            if ($container->hasAlias($serviceId)) {
```

Comment:
Should be replaced by something like [this](https://github.com/Ener-Getick/symfony/blob/795399a02bc68e404179c9f7ee7624715f1667b8/src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php#L1386-L1389) to work for any number of commands.